engine/admin/start-containers-automatically.md

```diff
@@ -78,3 +71,12 @@ the same `docker start` or `docker service` command you would normally use to
 start the container manually. Consult the documentation for the specific
 process manager for more details.
 
+### Using a process manager inside containers
```

Comment:
This is good guidance. I'd like to see it in an admonition, instead of an H3. I'd suggest using a warning class, as described in [Testing page](https://docs.docker.com/test/).

Reply:
Felt right to leave the h3 (just to distinguish between host and container process managers) and put a warning around the reason and the do not do.

Does that work for you? Or were you thinking the whole block (incl. the h3 in side an admonition?